global.json

```diff
@@ -5,7 +5,8 @@
       "dotnet": [
         "2.1.30",
         "$(MicrosoftNETCoreBrowserDebugHostTransportPackageVersion)",
-        "3.1.30"
+        "3.1.30",
+        "8.0.7"
```

Comment:
I'm wondering if this is needed then https://github.com/dotnet/razor/pull/10590/commits/9e8dbcc76afd98428935251f2cdd88d249d79600 or if the correct runtime will be installed in CI as well?

Reply:
I imagine it isn't needed.

Reply:
I'll try removing it and see if CI passes with this global.json tweak.

Reply:
@jjonescz: It looks like you were right. This is no longer needed.

Reply:
Thanks for the better fix!